src/wild_encounter.c

```diff
@@ -241,13 +268,19 @@ static u8 ChooseWildMonIndex_Fishing(u8 rod)
         else
             wildMonIndex = 1;
         break;
+
+		if (swap)
+			wildMonIndex = 1 - wildMonIndex;
```

Comment:
```suggestion
        if (swap)
            wildMonIndex = 1 - wildMonIndex;
```
Shouldn't this be put above the `break;`